charts/newrelic-k8s-metrics-adapter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
 name: newrelic-k8s-metrics-adapter
-version: 0.7.12
+version: 1.0.0
```

Comment:
a breaking change on a beta should not cause the bump of the major, otherwise it could seem that now it is GA

Reply:
we can say that the chart is GA while the product is beta @marcsanmi

Reply:
You're right, my bad.